apps/frontend/src/services/responses.ts

```diff
@@ -17,4 +17,18 @@ async function acceptResponse(responseId: string) {
   return response.data;
 }
 
-export default acceptResponse;
\ No newline at end of file
+async function deleteResponse(responseId: string) {
+  const headers: { authorization?: string } = {};
```

Comment:
There's a util for getting the logged user info in `auth.ts`

Reply:
you can use `getStoredUser` for these two functions, too

Reply:
Done!